.github/workflows/test.yaml

```diff
@@ -216,51 +216,6 @@ jobs:
           STDOUT_LOG=$(mktemp --suffix=stdout.log)
           echo STDOUT_LOG=$STDOUT_LOG >> $GITHUB_ENV
           tox --result-json=test-result.json | tee $STDOUT_LOG ; test ${PIPESTATUS[0]} -eq 0
-      - name: Check lint and test stdout
```

Comment:
What is the reason for removing this please?

Reply:
When using poetry, the dependencies will be handled in the `pyproject.toml` file and this test will always fail

Reply:
Could it somehow be adapted so that we still get the benefits? It is checking here that the expected tools are being run which is quite helpful to make sure that the expected linting is actually being done in a given repo